docs/source/notes/numerical_accuracy.rst

```diff
@@ -64,3 +64,11 @@ with fp32, however, if better accuracy is desired, TF32 can be turned off with
 ``torch.backends.cuda.matmul.allow_tf32 = False``
 
 For more information see :ref:`TensorFloat32<tf32_on_ampere>`
+
+Reduced Precision Reduction for FP16 GEMMs
+------------------------------------------
+Half-precision GEMM operations are typically done with intermediate accumulations (reduction) in single-precision for numerical accuracy and improved resilience to overflow. For performance, certain GPU architectures, especially more recent ones, enable reductions in reduced precision (e.g., half-precision). This change is often benign from the perspective of model convergence, though it may lead to unexpected results (e.g., ``inf`` values when the final result should be be representable in half-precision).
```

Comment:
```suggestion
Half-precision GEMM operations are typically done with intermediate accumulations (reduction) in single-precision for numerical accuracy and improved resilience to overflow. For performance, certain GPU architectures, especially more recent ones, allow a few truncations of the intermediate accumulation results to the reduced precision (e.g., half-precision). This change is often benign from the perspective of model convergence, though it may lead to unexpected results (e.g., ``inf`` values when the final result should be be representable in half-precision).
```